docs/containerd-env-setup.md

```diff
@@ -124,18 +126,28 @@ sudo systemctl restart containerd
 ```
 
 ### Using Runtime-level Snapshotter in Container for Nydus
-Containerd (>= v1.7.0) supports configuring the ```runtime-level``` snapshotter. By following the steps below, we can declare runtimes that use different snapshotters:
-#### Step 1: Configure Containerd
+
+Containerd (>= v1.7.0) supports configuring the `runtime-level` snapshotter. By following the steps below, we can declare runtimes that use different snapshotters:
```

Comment:
It's better to mention that the following patches are **only** needed if users want to use the `runtime-level` snapshotter feature, which was recently introduced and has some bugs.